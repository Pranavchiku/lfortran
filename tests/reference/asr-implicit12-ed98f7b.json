{
    "basename": "asr-implicit12-ed98f7b",
    "cmd": "lfortran --indent --show-asr --implicit-typing --no-color {infile} -o {outfile}",
    "infile": "tests/implicit12.f90",
    "infile_hash": "66a67ec98ce28bd0aaf5fbeddfa8b918e4c6e80df61f88fcdd2f38aa",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-implicit12-ed98f7b.stdout",
    "stdout_hash": "fb708434e97e9315dc1d76a0e7ddbc32fa16c6f55d18966ab94fef8b",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}