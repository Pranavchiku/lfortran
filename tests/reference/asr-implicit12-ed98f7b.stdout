(TranslationUnit 
    (SymbolTable
        1
        {
            a:
                (Function 
                    (SymbolTable
                        2
                        {
                            
                        }) 
                    a 
                    [] 
                    [] 
                    () 
                    Source 
                    Public 
                    Implementation 
                    () 
                    .false. 
                    .false. 
                    .false. 
                    .false. 
                    .false. 
                    [] 
                    [] 
                    .false.
                )
            
        }) 
    []
)
